cmd/trayscale/systray.go

```diff
@@ -1,6 +1,43 @@
 package main
 
-import "fyne.io/systray"
+import (
+	"fyne.io/systray"
+)
+
+type tray struct {
+	showItem *systray.MenuItem
+	quitItem *systray.MenuItem
+}
+
+func initTray(online bool) *tray {
+	systray.SetIcon(statusIcon(online))
+	systray.SetTitle("Trayscale")
+
+	showWindow := systray.AddMenuItem("Show", "")
+	systray.AddSeparator()
+	quit := systray.AddMenuItem("Quit", "")
+
+	return &tray{
+		showItem: showWindow,
+		quitItem: quit,
+	}
+}
+
+func (t *tray) NotfiyQuit() <-chan struct{} {
+	return t.quitItem.ClickedCh
+}
+
+func (t *tray) NotfiyShow() <-chan struct{} {
+	return t.showItem.ClickedCh
+}
+
+func (t *tray) SetIcon(online bool) {
+	if t == nil {
+		return
+	}
+
+	systray.SetIcon(statusIcon(online))
```

Comment:
Now that there's a more specific place for it, you could move `statusIcon()` and the two embeds it uses from `trayscale.go` into this file.